christianR15c/chat-room-back-end
# ch: setup project structure

# chore setup project structure

## What is about this PR
- setting up the project structure

## Why 
- to set up the entire project structure to follow for the development environment

## How to test
- clone this branch if you haven't done it yet
- checkout to this branch by typing this command `git checkout ch-project-structure-setup`
- install all dependencies by typing this command `npm install` or `npm i`
- open your code editor terminal and run `npm run dev` then you should be able to see that the server is running
